python/lsst/faro/measurement/VisitTableMeasurement.py

```diff
@@ -0,0 +1,48 @@
+import lsst.pipe.base as pipeBase
+from lsst.verify.tasks import MetricConnections
+import lsst.pex.config as pexConfig
+
+from lsst.faro.base.CatalogMeasurementBase import CatalogMeasurementBaseTaskConfig, CatalogMeasurementBaseTask
+
+__all__ = ("VisitTableMeasurementTaskConfig", "VisitTableMeasurementTask")
+
+
+class VisitTableMeasurementTaskConnections(MetricConnections,
+                                           dimensions=("instrument", "visit", "band")):
+
+    catalog = pipeBase.connectionTypes.Input(doc="Source catalog for visit.",
+                                             dimensions=("instrument", "visit", "band"),
+                                             storageClass="DataFrame",
+                                             name="sourceTable_visit",
+                                             deferLoad=True)
+
+    measurement = pipeBase.connectionTypes.Output(doc="Per-visit measurement.",
+                                                  dimensions=("instrument", "visit", "band"),
+                                                  storageClass="MetricValue",
+                                                  name="metricvalue_{package}_{metric}")
+
+
+class VisitTableMeasurementTaskConfig(CatalogMeasurementBaseTaskConfig,
+                                      pipelineConnections=VisitTableMeasurementTaskConnections):
+    columns = pexConfig.Field(doc="Columns from sourceTable_visit to load.",
+                              dtype=str, default='coord_ra, coord_dec')
+
+
+class VisitTableMeasurementTask(CatalogMeasurementBaseTask):
+    ConfigClass = VisitTableMeasurementTaskConfig
+    _DefaultName = "visitTableMeasurementTask"
+
+    def run(self, catalog, dataIds):
+        return self.measure.run(catalog, self.config.connections.metric, dataIds)
+
+    def runQuantum(self, butlerQC, inputRefs, outputRefs):
+        inputs = butlerQC.get(inputRefs)
+        columns = [_.strip() for _ in self.config.columns.split(',')]
```

Comment:
As John mentioned, this manipulation isn't necessary if you use a `ListField` in the config

Reply:
Done